### PR TITLE
initial support for TechNexion BSP layer

### PR DESCRIPTION
**Pull request by mirzak**
I have been thinking about if I should merge this together with `meta-mender-imx`, but in the end decided not to. Mostly because we will add a couple of more boards here soon and wanted to see how that turns out first, and could follow up with a consolidation.

### CODE REVIEW

File: meta-mender-tn-imx-bsp/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Quick start
+
+The following commands will setup the environment and allow you to build images
+that have Mender integrated.
+
+
+```
+mkdir mender-tn-imx-bsp && cd mender-tn-imx-bsp
+repo init -u https://github.com/TechNexion/tn-imx-yocto-manifest.git -b zeus_5.4.y-next -m imx-5.4.24-2.1.0.xml
+wget --directory-prefix .repo/manifests https://raw.githubusercontent.com/mendersoftware/meta-mender-community/zeus/meta-mender-tn-imx-bsp/scripts/imx-5.4.24-2.1.0_demo_mender.xml
```
Comment (drewmoseley) [inline meta-mender-tn-imx-bsp/README.md:27]: Should this go into .repo/local_manifests instead?  That's a standard repo feature.

Comment (mirzak) [reply meta-mender-tn-imx-bsp/README.md:27]: Do you know what to specify [this](https://github.com/mendersoftware/meta-mender-community/blob/7bffa1164bc79e77a313b5b4a00f4817f3078e72/meta-mender-tn-imx-bsp/scripts/imx-5.4.24-2.1.0_demo_mender.xml#L4) in that case?

Comment (mirzak) [reply meta-mender-tn-imx-bsp/README.md:27]: Or can that be omitted in that case?

Comment (mirzak) [reply meta-mender-tn-imx-bsp/README.md]: Indeed :), updated.

**Pull request merged**